### PR TITLE
prosody 0.12.1 lua 5.4 prosody.cfg.lua new gc values

### LINKED ISSUE #1392: prosody 0.12.1 lua 5.4 prosody.cfg.lua new gc values
https://community.jitsi.org/t/prosody-0-11-13-lua5-2-vs-0-12-1-lua-5-4-performance-difference/116785/12?u=krebya

### PR DESCRIPTION
**Pull request by ayberkkale**

### CODE REVIEW

**Comment by saghul** [comment]
Awesome! Can you please list the new variables in docker-compose.yml ?

File: prosody/rootfs/defaults/prosody.cfg.lua [hunk 1]
```diff
@@ -105,6 +111,21 @@ limits = {
   };
 }
 
+{{ if eq $GC_TYPE "generational" }}
```
Comment (saghul) [inline prosody/rootfs/defaults/prosody.cfg.lua:114]: maybe add a comment here with the link to the prosody site for more information.

Comment (ayberkkale) [reply prosody/rootfs/defaults/prosody.cfg.lua:116]: added

**Comment by ayberkkale** [comment]
> 



> 

added to yml

**Review by saghul — APPROVED** [review]

**Pull request merged**